.rubocop_todo.yml

```diff
@@ -12,11 +12,6 @@ Lint/AssignmentInCondition:
     - 'lib/openc_bot/helpers/incremental_search.rb'
     - 'lib/openc_bot/helpers/register_methods.rb'
 
-Lint/HandleExceptions:
```

Comment:
Yeah it would be better in a separate commit.

Thinking about it, the tasks seem to be related to the dockerised SRUs and any errors thrown would throw an error in the stdout output, except we have logic to check nothing was raised and then parse if result.success? so again, seemingly redundant.

For continuity sake I'll leave it as was, as I don't think it fits the scope of this PR. I want to run a code cleanup on the entire repo soon anyway, so I'll look at this type of stuff then.

Reply:
Would certainly be good to eliminate those unhandled exceptions, but yeah. Seems a bit thorny. Future PR maybe ✅